design-collaboration-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Design Collaboration Guidelines
+
+As an app developer to provide the best version of the design, you need to define a standard vocabulary between the design team and you.
+
+Designers are more and more integrating design systems as part of their work, and we can benefit from that to layout common ground. Here below we would define the minimum standard requirements to keep consistency and the usuals forgotten piece of app design. Independently of the size of your team, there is already a set of best practices applied by your designer that you can benefit, and other you will need anyway at some point during the development of the app.
+
+* [A COMPREHENSIVE GUIDE TO DESIGN SYSTEMS](https://www.invisionapp.com/blog/guide-to-design-systems/)
+
+## Style guide - Foundation
+These are the basic style that will serve as the foundation of all other components. And avoid mistakes by not using the proper element. Independently if you follow material design, fluent design or any other guidelines, these basics style element will facilitate design adjustment and rebranding.
+### Color Palette
+A list of all the color used in the app with a name associated.
+
+![color palette](./assets/color-palette.png)
+
+* [SwiftGen Colors](https://github.com/SwiftGen/SwiftGen#colors)
+* [Asset catalog colors on Xcode 9](https://blog.zeplin.io/asset-catalog-colors-on-xcode-9-c4fdccc0381a)
+* [Android Resources Color](https://developer.android.com/guide/topics/resources/more-resources.html#Color)
+* [Material Guidelines Color System](https://material.io/guidelines/style/color.html#color-color-system)
+
+### Typography - Fonts
+A list of Fonts and Size and when possible complete style including color and format.
+
+![typography](./assets/typography.png)
+
+### Layout - Sizing and Spacing
+Defining base templates that respect ratio and size of most used or more common devices that will be used. And a general rule for padding or margin that you can apply to most of the screen.
+
+![layout ios](./assets/layout-ios.png)
+
+![layout android](./assets/layout-android.png)
+
+## Components - Pattern library - Atoms
+Different design systems will have different naming for this, in Atomic Design you will call this Atoms, in others, it's just called Pattern Library, or Components. All these building blocks should be based style guide already provided and will be used to build more complex elements of the user interface like pages and screens.
+
+Commons components  you can find in most applications are:
+* Labels
+* Inputs
+* Buttons
```

Comment:
I'd add an example of the components more related to the business and not just the basic ones composed using the ones you list here 😃

Reply:
I will search for one.

Reply:
After thinking about it, I think it's out of scope to have business components, as these should be part of the design of each app and not listed in a generic guideline.